app/Item.php

```diff
@@ -565,7 +565,7 @@ public function getAuthorsWithLinks()
 
     public function getTitleWithAuthors($html = false)
     {
-        $dash = ($html) ? ' &ndash; ' : ' - ';
+        $dash = ($html) ? ' – ' : ' - ';
         return implode(', ', $this->authors)  . $dash .  $this->title;
```

Comment:
```suggestion
        return implode(', ', $this->authors)  . ' – ' .  $this->title;
```

I'd guess we don't even need the `$html` argument in this method?